cmd/calibrate/main.go

```
@@ -0,0 +1,235 @@
+	return out, nil
+}
+
+// writeParams 把标定结果写成 JSONL（路径或 stdout）.
+func writeParams(path string, params []datastat.ItemIrtStats) error {
+	var w io.Writer = os.Stdout
+	if path != "" {
+		f, err := os.Create(path)
+		if err != nil {
+			return err
+		}
+		defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)